3.5/security-security-options.md

```diff
@@ -40,38 +40,55 @@ an overview of the relevant options.
 Several options exists to restrict JavaScript application code functionality 
 to just certain allowed subsets. Which subset of functionality is available
 can be controlled via blacklisting and whitelisting access to individual 
-components. Blacklists can be used to disallow access to dedicated functionality, 
-whereas whitelists can be used to explicitly allow access to certain functionality.
+components.
 
-If an item is covered by both a blacklist and a whitelist, the whitelist will 
-overrule and access to the functionality will be allowed.
+The set thery for these list work as follow:
+
+- only a blacklist is specifiend: all is allowed except a set of items matching the blacklist.
+- only a whitelist is specified: all is disallowed except a set of items matching the whitelist
+- white and blacklist specified: all is disallowed except a set of items mathching the whitelist. From this whitelisted set, subsets can be forbidden again using blacklists.
```

Comment:
"white and blacklist specified: everything that matches a whitelist item and does not match a blacklist item is allowed."